python/erdos/operators/map.py

```diff
@@ -0,0 +1,17 @@
+class MapWindowOp(erdos.Operator):
+    """Operator transforms a given list of Messages using a user-defined function"""

```

Comment:
Since this is a generic map, let's update the docstring. Something like this would be good:
```python
"""Applies the provided function to a message and sends the resulting message."""
```